src/eth_rpc/servers/eth_rpc.rs

```diff
@@ -235,9 +241,21 @@ where
         Err(EthApiError::Unsupported("eth_sendTransaction").into())
     }
 
+    #[allow(unreachable_code)]
```

Comment:
why do we need this?

Reply:
Just to avoid a trunk error, since one return makes the other unreachable, trunk gives me an "unreachable expression" error.

Reply:
my opinion is that it's nicer to do 
```rust
#[cfg(feature = "my_feature")]
{
}
#cfg(not(feature = "my_feature"))]
{
}
```
This way you avoid the unreachable and the code is clearly split in two "zones"